installer/mod.ts

```diff
@@ -140,15 +166,57 @@ async function fetchModule(url: string): Promise<any> {
 function showHelp(): void {
   console.log(`deno installer
   Install remote or local script as executables.
-  
+
 USAGE:
-  deno https://deno.land/std/installer/mod.ts EXE_NAME SCRIPT_URL [FLAGS...]  
+  deno run https://deno.land/std/installer/mod.ts EXE_NAME SCRIPT_URL [FLAGS...]
 
 ARGS:
-  EXE_NAME  Name for executable     
+  EXE_NAME  Name for executable
   SCRIPT_URL  Local or remote URL of script to install
   [FLAGS...]  List of flags for script, both Deno permission and script specific flag can be used.
-  `);
+`);
+}
+
+async function genereateExecutable(
+  filePath: string,
+  commands: string[]
+): Promise<void> {
+  // genereate Batch script
+  if (isWindows) {
+    const template = `% This executable is generated by Deno. Please don't modify it unless you know what it means. %
+@IF EXIST "%~dp0\deno.exe" (
+  "%~dp0\deno.exe" ${commands.slice(1).join(" ")} %*
+) ELSE (
+  @SETLOCAL
+  @SET PATHEXT=%PATHEXT:;.TS;=;%
+  ${commands.join(" ")} %*
+)
+`;
+    const cmdFile = filePath + ".cmd";
```

Comment:
Shouldn't this be done in `install` on this line: `const filePath = path.join(installerDir, moduleName);`?

Reply:
see here https://github.com/denoland/deno_std/pull/499#discussion_r294624755

Reply:
Ok, makes sense :+1: can you add a comment in the code for that?

Reply:
sure.